sender.py

```diff
@@ -3,7 +3,7 @@
 import json
 import logging
 import socket
-import struct
+import struct #Sample
```

Comment:
What is this supposed to do? Doesn't seem to change anything. Is that some kind of special comment?